tests/integration/product-type-import.spec.js

```diff
@@ -0,0 +1,174 @@
+import { SphereClient } from 'sphere-node-sdk'
+import Promise from 'bluebird'
+import test from 'tape'
+import ProductTypeImport from '../../src'
+import getSphereClientCredentials from '../../src/utils'
+
+const PROJECT_KEY = 'sphere-node-product-type-import'
+/* eslint-disable no-console */
+const logger = {
+  trace: console.log,
+  debug: console.log,
+  info: console.log,
+  error: console.error,
+}
+/* eslint-enable no-console */
+const deleteAll = (service, client) => client[service].process(
+  ({ body: { results } }) => Promise.map(
+    results, productType => client[service].byId(
+      productType.id
+    ).delete(
+      productType.version
+    )
+  )
+)
+
+let client
+let productTypeImport
+
+const before = function setupSphereCreds () {
+  return getSphereClientCredentials(PROJECT_KEY)
+  .then((sphereCredentials) => {
+    const options = {
+      config: sphereCredentials,
+    }
+    client = new SphereClient(options)
+
+    productTypeImport = new ProductTypeImport(
+      logger,
+      { sphereClientConfig: options }
+    )
+    return deleteAll('productTypes', client)
+  })
+}
```

Comment:
This all setup can be simplified after we refactor the way of reading test credentials